app/javascript/components/patient/laboratory/LaboratoryModal.js

```diff
@@ -186,12 +272,26 @@ class LaboratoryModal extends React.Component {
   }
 }
 
+var schema = yup.object().shape({
+  lab_type: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
+  specimen_collection: yup
+    .date('Date must correspond to the "mm/dd/yyyy" format.')
+    .min(moment('2020-01-01'), 'Specimen Collection Date must fall after January 1, 2020.')
+    .max(moment(), 'Specimen Collection Date can not be in the future.')
+    .nullable(),
+  report: yup
+    .date('Date must correspond to the "mm/dd/yyyy" format.')
+    .min(moment('2020-01-01'), 'Report Date must fall after January 1, 2020.')
+    .max(moment(), 'Report Date can not be in the future.')
+    .nullable(),
+  result: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
```

Comment:
not sure if this needs to be validated since it's just a select input, but if it is, I think it should just be `oneOf` the enum values

Reply:
This is what we do across the board for any select field (see any of the ones in enrollment), so I'd prefer to keep these in here

Reply:
fair enough